modules/virt-creating-nad-l2-overlay-console.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-creating-nad-l2-overlay-console_{context}"]
+= Creating a NAD for layer 2 topology using the web console
+
+You can create a network attachment definition (NAD) which describes how to attach a pod to the layer 2 overlay network.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Navigate to *Networking* -> *NetworkAttachmentDefinitions* in the web console.
```

Comment:
```suggestion
. Go to *Networking* -> *NetworkAttachmentDefinitions* in the web console.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#n
See this link for adding backticks to system items (NetworkAttachmentDefinitions): https://redhat-documentation.github.io/asciidoc-markup-conventions/

Reply:
in this case, the `NetworkAttachmentDefinitions` is actually the UI element (the name of the link off to the left sidebar) even though it definitely looks like a system item. so i think we can keep it in bold here. correct me if i'm wrong please!